pkg/repository/repository.go

```diff
@@ -682,11 +682,11 @@ func (r *repository) transpileFilter(filter filtering.Filter, tableName string)
 	}).Transpile()
 }
 
-func (r *repository) ListModelTriggers(ctx context.Context, pageSize, page int64, filter filtering.Filter, order ordering.OrderBy, modelUID string) (modelTriggers []*datamodel.ModelTrigger, totalSize int64, err error) {
+func (r *repository) ListModelTriggers(ctx context.Context, pageSize, page int64, filter filtering.Filter, order ordering.OrderBy, dbModel *datamodel.Model, userUID string) (modelTriggers []*datamodel.ModelTrigger, totalSize int64, err error) {
```

Comment:
considering the below comment, we should pass in dbModel's UID and visibility instead of the whole object, and also a boolean value maybe called `isOwner`

Reply:
fixed